test/Microsoft.AspNetCore.SignalR.Redis.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -0,0 +1,264 @@
+                {
+                    Factory = t => new TestConnectionMultiplexer()
+                }));
+                var connection1 = new HubConnectionContext(output1, client1.Connection);
+                var connection2 = new HubConnectionContext(output2, client2.Connection);
+
+                await manager.OnConnectedAsync(connection1);
+                await manager.OnConnectedAsync(connection2);
+
+                await manager.OnDisconnectedAsync(connection2);
+
+                await manager.InvokeAllAsync("Hello", new object[] { "World" });
+
+                AssertMessage(output1);
+
+                Assert.False(output2.In.TryRead(out var item));
+            }
+        }
+
+        [Fact]
+        public async Task InvokeGroupAsyncWritesToAllConnectionsInGroupOutput()
+        {
+            using (var client1 = new TestClient())
+            using (var client2 = new TestClient())
+            {
+                var output1 = Channel.CreateUnbounded<HubMessage>();
+                var output2 = Channel.CreateUnbounded<HubMessage>();
+
+                var manager = new RedisHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<RedisHubLifetimeManager<MyHub>>(), Options.Create(new RedisOptions()
+                {
+                    Factory = t => new TestConnectionMultiplexer()
+                }));
+                var connection1 = new HubConnectionContext(output1, client1.Connection);
+                var connection2 = new HubConnectionContext(output2, client2.Connection);
+
+                await manager.OnConnectedAsync(connection1);
+                await manager.OnConnectedAsync(connection2);
+
+                await manager.AddGroupAsync(connection1.ConnectionId, "gunit");
+
+                await manager.InvokeGroupAsync("gunit", "Hello", new object[] { "World" });
+
+                AssertMessage(output1);
+
+                Assert.False(output2.In.TryRead(out var item));
+            }
+        }
+
+        [Fact]
+        public async Task InvokeConnectionAsyncWritesToConnectionOutput()
+        {
+            using (var client = new TestClient())
+            {
+                var output = Channel.CreateUnbounded<HubMessage>();
+                var manager = new RedisHubLifetimeManager<MyHub>(new LoggerFactory().CreateLogger<RedisHubLifetimeManager<MyHub>>(), Options.Create(new RedisOptions()
+                {
+                    Factory = t => new TestConnectionMultiplexer()
+                }));
+                var connection = new HubConnectionContext(output, client.Connection);
+
+                await manager.OnConnectedAsync(connection);
+
+                await manager.InvokeConnectionAsync(connection.ConnectionId, "Hello", new object[] { "World" });
+
+                AssertMessage(output);
+            }
+        }
+
+        [Fact]
+        public async Task InvokeConnectionAsyncOnNonExistentConnectionNoops()
```

Comment:
This test is a bit weird - the only thing you check here is really that it does not throw and not that it no ops.

Reply:
It is the exact same in DefaultHubLifetimeManagerTests but I'll change it to say DoesNotThrow